cranelift-simplejit/src/backend.rs

```diff
@@ -21,17 +21,23 @@ use winapi;
 pub struct SimpleJITBuilder {
     isa: Box<TargetIsa>,
     symbols: HashMap<String, *const u8>,
+    libcall_names: Box<Fn(ir::LibCall) -> String>,
 }
 
 impl SimpleJITBuilder {
     /// Create a new `SimpleJITBuilder`.
-    pub fn new() -> Self {
+    ///
+    /// The `libcall_names` function provides a way to translate `cranelift_codegen`'s `ir::LibCall`
+    /// enum to symbols. LibCalls are inserted in the IR as part of the legalization for certain
+    /// floating point instructions, and for stack probes. If you don't know what to use for this
+    /// argument, use `FaerieBuilder::default_libcall_names()`.
```

Comment:
In this comment and the one below, do you mean `SimpleJITBuilder::default_libcall_names()` instead?